src/main/java/in/zapr/druid/druidry/virtualColumn/DruidVirtualColumn.java

```diff
@@ -0,0 +1,18 @@
+package in.zapr.druid.druidry.virtualColumn;
+
+import in.zapr.druid.druidry.dimension.enums.OutputType;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@EqualsAndHashCode
+public abstract class DruidVirtualColumn {
+    @NonNull
+    protected String type;
+
+    @NonNull
+    protected String name;
+
+    protected OutputType outputType;
```

Comment:
Current `OutputType` enum only has `STRING, LONG, FLOAT`, can you add `DOUBLE` to this enum because expression virtual column supports `LONG, FLOAT, DOUBLE, or STRING`